### ISSUE TITLE
Timeouts in overviews process are not rescued and might create tables not fully setup

### ISSUE THREAD

**Issue by iriberri** [CartoDB/cartodb#8129]
* The canonical map of a table is created in the `after_create` fallback. See: https://github.com/CartoDB/cartodb/blob/73c9d1aa0d88d48ac65a34ce1e96f5f30a8d801b/app/models/table.rb#L404

* The table gets created inside the `register` function in the import process. See: https://github.com/CartoDB/cartodb/blob/73c9d1aa0d88d48ac65a34ce1e96f5f30a8d801b/app/models/table_registrar.rb#L11

* As an exception in the `create_overviews` method is not rescued, this piece of code (`store_results`) is never run: https://github.com/CartoDB/cartodb/blob/a58af3f553cc48380e5fee1d110567aa538e025b/app/models/data_import.rb and it seems that [this piece of code](https://github.com/CartoDB/cartodb/blob/a58af3f553cc48380e5fee1d110567aa538e025b/app/models/data_import.rb#L712) could be key for the correct map_id setup.

* This is the last method which is raising the exception, inside Data Import: https://github.com/CartoDB/cartodb/blob/master/app/models/data_import.rb#L390-L398

If you need to reproduce the issue, I have a table for it, just let me know!

**Comment by iriberri** [comment]
One more detail I noticed. In the dashboard, the table doesn't seem to have geometry type even though the table I'm uploading has a the_geom column which I think it's okay (it gets cartodbfied perfectly):

![image](https://cloud.githubusercontent.com/assets/1730320/16311661/b95cab9c-3971-11e6-8d22-ce04add91a38.png)

If you need it you can actually look at the table named `cositas` (no laughing ;D) in my db.

**Comment by jgoizueta** [comment]
For the table that failed it seems that `in2csv` appended over 1M blank rows to an input file of less than 3K.

This appears in the import trace:

```
true: Converting XLSX to CSV
true: About to execute in2csv: in2csv /tmp/imports/.../cositas.xlsx | grep -v "WARNING \*\*\* OLE2 inconsistency: SSCS size is 0 but SSAT size is non-zero" | grep -v "\*\*\* No CODEPAGE record, no encoding_override: will use 'ascii'" | /usr/bin/python /home/ubuntu/www/production.cartodb.com/releases/20160622151212/lib/importer/misc/csv_remove_newlines.py > /tmp/imports/../cositas.csv
true: done executing in2csv.
```

I'll move this to a separate ticket and we'll deal with handling overview creation failures gracefully here

**Comment by jgoizueta** [comment]
Basically we can take two different approaches:

* Ignore (but log) any error during overviews creation (or at least timeout errors) and proceed with a successful import without overviews. This may seam reasonable since overviews are not essential.
* Abort the import (with proper cleanup) and mark it as failed. This might actually be better in cases as the one we have found since it there exist an undetected problem (the extra blank rows) that caused the overviews timeout.

I'm proceeding with the first approach.

**Comment by jgoizueta** [comment]
The problem of overview creation fails leaving tables in a bad state has been fixed by #8169 
and now fails can be easily detected in logs.

The tables that were causing the timeouts will be considered in #8500
